index.js

```diff
@@ -1,34 +1,31 @@
 const dotenv = require('dotenv');
 dotenv.config();
 process.env.MONGO_URL;
-
-const { MongoClient } = require('mongodb');
-
 const express = require('express');
-const path = require('path');
-
 const port = process.env.PORT || 8080;
 const app = express();
+const path = require('path');
+const mongoose = require('mongoose');
+const authRoute = require('./lib/routes/auth');
```

Comment:
```suggestion
const authRouter = require('./lib/routes/auth');
```

or 

```suggestion
const authRoutes = require('./lib/routes/auth');
```

same for `postRoute`